api/src/main/java/org/accula/api/handlers/request/RequestBody.java

```diff
@@ -0,0 +1,15 @@
+package org.accula.api.handlers.request;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.ALWAYS;
+
+/**
+ * @author Anton Lamtev
+ */
+@JsonInclude(ALWAYS)
+@JsonAutoDetect(fieldVisibility = ANY)
+public interface RequestBody {
```

Comment:
Make the name of the interface more self-explanatory. See comment for ResponseBody below

Reply:
__-__